alf/optimizers/nero_plus_test.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2022 Horizon Robotics and ALF Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from absl.testing import parameterized
+from absl import logging
+import torch
+import torch.nn.functional as F
+
+import alf
+
+from alf.optimizers import NeroPlus
+from alf.utils.datagen import load_mnist
+
+
+class NeroPlusTest(alf.test.TestCase):
+    def test_nero_plus(self):
+        train_set, test_set = load_mnist(train_bs=256, test_bs=256)
+        num_classes = len(train_set.dataset.classes)
+        model = alf.layers.Sequential(
+            alf.layers.Conv2D(1, 32, 3, strides=2, padding=1),
+            alf.layers.Conv2D(32, 32, 3, strides=2, padding=1),
+            alf.layers.Conv2D(32, 32, 3, strides=2, padding=1),
+            alf.layers.Reshape(-1),
+            alf.layers.FC(
+                4 * 4 * 32,
+                num_classes,
+                weight_opt_args=dict(
+                    fixed_norm=False, weight_decay=1e-3,
+                    max_norm=float('inf'))))
+        NeroPlus.initialize(model)
+        opt = NeroPlus(lr=0.01)
```

Comment:
Do you want to also include test cases beyond the default setting, say original Nero and without running average?

Reply:
added